prompts/long_prompt_long_output.txt

```diff
@@ -60,4 +63,6 @@ You must return the following:
 - Include a detailed list of materials and specifications.
 - Add a cover sheet with project title, address, date, and designer's name.
 - Add a sheet for each component with detailed plans.
-- Ensure all documents are clearly labeled and organized.
\ No newline at end of file
+- Ensure all documents are clearly labeled and organized.
+
+<|gist|>### Response:
```

Comment:
These are sample prompts for any model -- can we incorporate `<|gist|>` logic into `tokenizer.py`?